packages/graphql/src/services/courses.ts

```diff
@@ -119,6 +150,10 @@ export async function leaveCourse(
     },
   })
 
+  await ctx.prisma.leaderboardEntry.deleteMany({
```

Comment:
delete leaderboard entries corresponding to the current participation when a course is being left (means that all points from any activity are deleted)